internal/bitcoin/bitcoin.go

```diff
@@ -33,9 +33,9 @@ type BitcoinDto struct {
 	sharedAPI.Resources
 }
 
-type BitcoinListDto []BitcoinDto
+type BitcoinListDto []*BitcoinDto
 
-func (dto BitcoinDto) FromBitcoinNode(n *bitcoinv1alpha1.Node) *BitcoinDto {
+func (dto BitcoinDto) FromBitcoinNode(n bitcoinv1alpha1.Node) *BitcoinDto {
```

Comment:
return struct instead of pointer to a struct, because we're modifying a receiver.

Do the same for all protocols.